### ISSUE TITLE
Termux vim-python : Error detected while compiling CursorHold Autocommands for "*"

### ISSUE THREAD

**Issue by cainslayer** [amix/vimrc#690]
After I install the `basic.vim` script in termux, this error message shows up every few seconds:

```
Error detected while compiling CursorHold Autocommands for "*"..function <SNR>14_SetupLazyloadMenus[1]..<SNR>14_SetupColorSchemes:        line   11:                                    E1256: String or function required for argument 2
```

Sorry that I know nothing about vim script. But with `finish` command, I knew that if I comment out these lines, the message will disappear.

```
" Avoid garbled characters in Chinese
 language windows OS
 let $LANG='en'
 set langmenu=en
 source $VIMRUNTIME/delmenu.vim
 source $VIMRUNTIME/menu.vim
```

I was using this `basic.vim` for a long time, and this problem happens lately. I don't know if termux or vim or other who causes the problem(I did `apt purge vim-python` and removed all .vim* in home directory but it doesn't take effects). Did someone meet the problem?

**Comment by alexandergazo** [comment]
I have encountered the problem on two separate machines. Exact same error message and python.

**Comment by urod27** [comment]
I am also having this issue. How do you fix it?

**Comment by cainslayer** [comment]
> I am also having this issue. How do you fix it?

I commented out following 4 lines in `~/.vim_runtime/vimrcs/basic.vim`, the problem occurs much less.

```
" Avoid garbled characters in Chinese
 language windows OS
 let $LANG='en'
 set langmenu=en
 source $VIMRUNTIME/delmenu.vim
 source $VIMRUNTIME/menu.vim
```

As @alexandergazo mentioned, the 'menu.vim' line causes the problem. I found I can reproduce the problem with only two lines:

```
source $VIMRUNTIME/delmenu.vim
source $VIMRUNTIME/menu.vim
```

Although I removed the old `.vimrc` (Maybe it is equivalent to uninstalling this software?). A new `.vimrc` with only those 2 lines can produce the problem too.

Those 2 lines is a normal functionality of vim, they shouldn't produce errors without any customized modification. Hence I believe the problem is caused by termux vim-python, rather than this software.
